htdocs/install/sql/mysql.structure.sql

```diff
@@ -462,7 +462,7 @@ CREATE TABLE session (
   sess_updated int(10) unsigned NOT NULL default '0',
   sess_ip varchar(45) NOT NULL default '',
   sess_data text,
-  PRIMARY KEY  (sess_id),
+  PRIMARY KEY  (sess_id(200)),
```

Comment:
Subsetting a primary key -- yikes!

This should not be necessary, as sess_id is declared ascii, not UTF8, so each index character is only a single byte. This column declaration is matched to the PHP maximum possible session id characteristics, 

I've verified this works without the 200 prefix length change in MySQL 8.0.28. If you have an environment where the full length is a problem, please describe it.

Reply:
I was installing it on PHP 8.2 RC and MySQL 5.7.39, and was getting errors. But it's possible that it was related to something else. I'll keep testing, but for the time being I'll reverse it.